src/EnergyPlus/WindTurbine.cc

```diff
@@ -278,15 +285,10 @@ namespace WindTurbine {
             }
 
             // Select control type
-            {
-                auto const SELECT_CASE_var(state.dataIPShortCut->cAlphaArgs(4));
-                if (SELECT_CASE_var == "FIXEDSPEEDFIXEDPITCH") {
-                    state.dataWindTurbine->WindTurbineSys(WindTurbineNum).controlType = ControlType::FSFP;
-                } else if (SELECT_CASE_var == "FIXEDSPEEDVARIABLEPITCH") {
-                    state.dataWindTurbine->WindTurbineSys(WindTurbineNum).controlType = ControlType::FSVP;
-                } else if (SELECT_CASE_var == "VARIABLESPEEDFIXEDPITCH") {
-                    state.dataWindTurbine->WindTurbineSys(WindTurbineNum).controlType = ControlType::VSFP;
-                } else if ((SELECT_CASE_var == "VARIABLESPEEDVARIABLEPITCH") || (SELECT_CASE_var == "")) {
+            state.dataWindTurbine->WindTurbineSys(WindTurbineNum).controlType =
```

Comment:
An idiom I am suggesting to everyone in as many places as I can is:

```cpp
auto &windTurbine = state.dataWindTurbine->WindTurbineSys(WindTurbineNum);
windTurbine.controlType = ...
if (windTurbine.controlType == ...) 
...
```
Much cleaner and fewer confusing wraparound lines.

Reply:
Thank you. Will make the change.

Reply:
Making this change at other places in the file as well, where possible.

Reply:
I'm not a huge fan of references (just use pointers, why the disguise?) and certainly not of `auto`. But I think using  `auto &` to create local variables that can shorten code (especially if it prevents line-wrapping) is a good thing. Especially helpful now that we have `state.dataXXX->` in front of most things.